marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/ExecEndpointImpl.java

```diff
@@ -145,19 +148,21 @@ public void setErrorListener(ErrorListener errorListener) {
             this.errorListener = errorListener;
         }
 
-        private void processOutput() {
-            CallContextImpl callContext = getCallContext();
-            if(callContext != null) {
-                while (processOutput(callContext));
-            }
+        private ExecEndpoint.BulkExecCaller.ErrorListener getErrorListener() {
+            return this.errorListener;
         }
 
-        private boolean processOutput(CallContextImpl callContext){
-                InputStream output = null;
+        private InputStream processExec(CallContextImpl callContext) {
+            final int DEFAULT_MAX_RETRIES = 10;
```

Comment:
Is there a reason that DEFAULT_MAX_RETRIES is 10 in some callers and 100 in others?

Could the constant be shared across the implementations if declared as a static final package-visible constant of IOEndpointImpl?

Reply:
Sorry, that's a number set for testing. Added below statement in IOEndpointImpl.java

final static int DEFAULT_MAX_RETRIES = 100;